blueprints/tools/blueprint_crs_converter.py

```diff
@@ -81,11 +81,19 @@ def crs_converter_missing_files():
     mandatory_files = missing_files.mandatory_files
     additional_files = missing_files.additional_files
 
+    mandatory_files_list = []
+    for mandatory_file in mandatory_files:
+        mandatory_files_list.append(os.path.basename(mandatory_file))
```

Comment:
Should this be done on c++ side?

Reply:
Maybe, I don't see the point orfreturning paths